jscripts/usercp.js

```diff
@@ -199,7 +199,7 @@ var UserCP = {
 			type: 'post',
 			url: 'usercp.php?action=do_editlists&my_post_key='+my_post_key+'&manage='+type,
 			data: { ajax: 1, add_username: type_add_username.val() },
-			async: false,
+			async: true,
```

Comment:
Is this related to the update?

Reply:
It started throwing console warning after update, hence changed.

Reply:
It could be removed completely since `true` is default (https://api.jquery.com/jQuery.ajax/)

Reply:
Re declaring the default value doesn't harm a thing. But I will remove those; if you insist.